Components/ListSearchResults/Partials/Header/_style.scss

```diff
@@ -50,14 +49,11 @@ $search-form-spacing: 6px;
     margin: 0;
     min-height: 44px;
     outline: none;
-    padding: 0 1.25rem 0 3.125rem;
-    transition: all 0.1s cubic-bezier(0.175, 0.885, 0.32, 0.275);
+    padding: 0 1.25rem 0 (20px - 6); // .boxPadding (mobile) - $search-form-spacing
```

Comment:
I feel all margins should be in pixels then, mixed is confusing to me. Also, why don't you just use the actual variables instead of commenting them?

Reply:
Padding was originally meant to be expressed in `rem`, but since the side spacing needs to follow the `.boxPadding` values which are expressed in `px`. 
I will refactor the padding to `px` and introduce a variable that takes care of the box spacing.

Reply:
Ah, I thought we had a variable for `.boxPadding` already. I don't think we need to introduce one then. Would be fine with converting everything to px instead and just using the `$search-form-spacing` variable that's available already.